diff -up otf2-1.5.1/build-backend/configure.ac.autoconf otf2-1.5.1/build-backend/configure.ac
--- otf2-1.5.1/build-backend/configure.ac.autoconf	2014-12-09 08:14:07.000000000 -0700
+++ otf2-1.5.1/build-backend/configure.ac	2015-02-11 16:44:33.586014839 -0700
@@ -37,8 +37,7 @@ AC_SUBST([LIBRARY_INTERFACE_VERSION], m4
 AC_DEFINE_UNQUOTED([LIBRARY_INTERFACE_VERSION], ["$LIBRARY_INTERFACE_VERSION"], [Interface version number])
 
 AC_CONFIG_AUX_DIR([../build-config])
-AC_CONFIG_MACRO_DIR([../build-config/m4
-                     ../vendor/common/build-config/m4])
+AC_CONFIG_MACRO_DIR([../build-config/m4])
 
 AC_CONFIG_HEADERS([../src/config-backend.h])
 AS_IF([test "x${ac_scorep_cross_compiling}" = "xyes"],
diff -up otf2-1.5.1/build-frontend/configure.ac.autoconf otf2-1.5.1/build-frontend/configure.ac
--- otf2-1.5.1/build-frontend/configure.ac.autoconf	2014-12-09 08:14:09.000000000 -0700
+++ otf2-1.5.1/build-frontend/configure.ac	2015-02-11 16:42:40.250536440 -0700
@@ -37,8 +37,7 @@ AC_SUBST([LIBRARY_INTERFACE_VERSION], m4
 AC_DEFINE_UNQUOTED([LIBRARY_INTERFACE_VERSION], ["$LIBRARY_INTERFACE_VERSION"], [Interface version number])
 
 AC_CONFIG_AUX_DIR([../build-config])
-AC_CONFIG_MACRO_DIR([../build-config/m4
-                     ../vendor/common/build-config/m4])
+AC_CONFIG_MACRO_DIR([../build-config/m4])
 
 AC_CONFIG_HEADERS([../src/config-frontend.h])
 
diff -up otf2-1.5.1/configure.ac.autoconf otf2-1.5.1/configure.ac
--- otf2-1.5.1/configure.ac.autoconf	2014-12-09 08:14:09.000000000 -0700
+++ otf2-1.5.1/configure.ac	2015-02-11 16:42:40.250536440 -0700
@@ -45,10 +45,7 @@ AC_SCOREP_TOPLEVEL_ARGS
 ## in this directory.
 AC_CONFIG_AUX_DIR([build-config])
 
-## Packages that use aclocal should declare where local macros can be found
-## using AC_CONFIG_MACRO_DIR.
-AC_CONFIG_MACRO_DIR([build-config/m4
-                     vendor/common/build-config/m4])
+AC_CONFIG_MACRO_DIR([build-config/m4])
 
 ## Set the default installation prefix to /opt/otf2 instead of /usr/local
 AC_PREFIX_DEFAULT([/opt/otf2])
@@ -144,3 +141,5 @@ AC_CONFIG_FILES(
 AC_OUTPUT
 
 AFS_SUMMARY_COLLECT([test "x$ac_scorep_platform_data_provided" != "xyes"])
+
+LT_INIT
