diff -up otf2-1.4/Makefile.am.jinja2 otf2-1.4/Makefile.am
--- otf2-1.4/Makefile.am.jinja2	2014-05-12 03:47:30.000000000 -0600
+++ otf2-1.4/Makefile.am	2014-07-15 13:58:51.126064624 -0600
@@ -139,7 +139,6 @@ src/otf2_error_decls.gen.h: $(srcdir)/sh
 
 include share/Makefile.inc.am
 include src/tools/otf2_template/Makefile.inc.am
-include vendor/python/Makefile.inc.am
 
 include templates/Makefile.inc.am
 
